src/Sulu/Bundle/ContentBundle/Resources/js/views/WebspaceOverview/WebspaceOverview.js

```diff
@@ -14,11 +17,45 @@ class WebspaceOverview extends React.Component<ViewProps> {
     locale: IObservableValue<string> = observable();
     webspace: IObservableValue<string> = observable();
     datagridStore: DatagridStore;
+    @observable webspaces: Array<Webspace>;
 
     @action handleWebspaceChange = (value: string) => {
         this.webspace.set(value);
+        this.setDefaultLocaleForWebspace();
     };
 
+    @action setDefaultLocaleForWebspace = () => {
+        if (!this.selectedWebspace) {
+            return;
+        }
+
+        const localization = this.selectedWebspace.localizations.find((localizations) => {
+            if (localizations.children) {
+                localizations.children.find((localizations) => {
```

Comment:
Does this also work more than two levels deep? I think this should be called recursive to work properly...